keymanapp/keyman
# chore(common): Add crowdin for Swedish

Adds crowdin strings for Swedish for locale sv-SE
Using localized name `svenska` from https://github.com/unicode-org/cldr/blob/a88558301989b98d5bedb13e9123ec6d12f82416/common/main/sv.xml#L577

## User Testing
This adds the crowdin strings for Swedish locale sv-SE

TODO for @sgschantz 

## User Testing
Load the PR build for each platform and set the UI / Locale to "svenska" (Swedish)

* **TEST_ANDROID** 
1. Load the PR build of Keyman for Android
2. In the Keyman app, go to Settings --> Display Language --> svenska (Swedish)
3. Return to the Keyman app 
4. Verify the UI is in Swedish (prompt, Settings strings, etc.).



* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the device locale to Swedish
3. Return to the Keyman app
4. Verify the UI is in  Swedish

* **TEST_LINUX**
1. Start Keyman configuration with: LANGUAGE=sv_SE km-config
2. Verify Keyman UI is in Swedish

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the locale to Swedish
3. Verify the Keyman UI is in Swedish

* **TEST_WINDOWS**
1. Load the PR build of Keyman for Windows
2. Set the UI to svenska (Swedish)
3. Verify the Keyman UI is in Swedish
